examples/contrib/breakhis/model/breakhis.py

```diff
@@ -1,3 +1,18 @@
+############################################################################
```

Comment:
Radha:

Do we really need this ?  As this contrib and not HPE developed ?  What's the legal implication ?

Reply:
Ignore